deployer/src/lib.rs

```diff
@@ -452,6 +453,21 @@ impl StartOptions {
             0
         }
     }
+
+    /// Get the io-engine api versions.
+    pub fn io_api_versions(&self) -> &Vec<IoEngineApiVersion> {
+        &self.io_engine_api_versions
+    }
+    /// Get the latest io-engine api version.
+    pub fn latest_io_api_version(&self) -> IoEngineApiVersion {
+        let mut sorted = self.io_engine_api_versions.clone();
+        sorted.sort();
+        if !sorted.is_empty() {
```

Comment:
Do we need this check? We will always have a value right because of the default value ?

Reply:
We don't need it, that was me trying to return by value instead of reference, but it doesn't matter here I guess.

Reply:
Done